modules/installing-rhv-requirements.adoc

```diff
@@ -2,42 +2,48 @@
 //
 // * installing/installing_rhv/installing-rhv-custom.adoc
 // * installing/installing_rhv/installing-rhv-default.adoc
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
 
 [id="installing-rhv-requirements_{context}"]
 = Requirements for the {rh-virtualization} environment
 
-To install and run an {product-title} cluster, the {rh-virtualization} environment must meet the following requirements. Not meeting these requirements can cause failures.
+To install and run an {product-title} cluster, the {rh-virtualization} environment must meet the following requirements.
 
-The following requirements for CPUs, memory, and storage are based on _default_ values multiplied by the default number of virtual machines the installation program creates.
+Not meeting these requirements can cause the installation or process to fail. Additionally, not meeting these requirements can cause the {product-title} cluster to fail days or weeks after installation.
 
-By default, the installation program creates seven machines during the installation process, which includes one bootstrap machine. When the installation program finishes, it deletes the bootstrap machine and frees up its resources. If you perform a custom installation, you can increase the number of virtual machines the installation program creates.
+The following requirements for CPU, memory, and storage resources are based on _default_ values multiplied by the default number of virtual machines the installation program creates. These resources must be available **aside from** what the {rh-virtualization} environment uses for non-{product-title} operations.
 
-[IMPORTANT]
-====
-If you increase the number of virtual machines or resources in the `install_config.yaml` file, you must also increase these requirements.
-====
+By default, the installation program creates seven virtual machines during the installation process. First, it creates a bootstrap virtual machine to provide temporary services and a control plane while it creates the rest of the {product-title} cluster. When the installation program finishes creating the cluster, it deletes the bootstrap machine to free up its resources.
```

Comment:
Is this true? Don't you manually have to delete the bootstrap?

Reply:
Thanks for catching issues like this. I scrubbed this shared module for IPI-specific statements like this one. With UPI, the user removes the bootsrap machine at the end of the process.